resources/views/projects/details.blade.php

```diff
@@ -90,7 +90,74 @@
 					@endforeach
 				</div>
 			</div>
+			<!-- Progress section -->
+			<div class="sixteen wide column">
+				<div class="ui left aligned detail-container">
+					<p><strong>Progress</strong></p>
+					<div class="ui blue progress">
+						<div class="bar">
+							<div class="progress"></div>
+						</div>
+						<div class="label">Currently on: Design</div>
+					</div>
+					<div class="ui right floated button" onclick="displayMilestones()">
+						<a class="milestones" title="See project's milestones">See milestones</a>
+					</div>
+					<div class="ui list milestone-section hidden">
```

Comment:
Me encanta cómo se ve con la lista! 

Siento que está un poquito cluttered en el overview so here's a suggestion and it's entirely up to you: como la lista puede seguir creciendo ad infinitum, podríamos considerar ver la lista en su propia tab de milestones vs en el overview... it's a trade-off porque si son poquitas igual y "gastamos" el espacio, entonces como tú veas mejor.

Reply:
Si lo pensé, pero no estaba segura...
How about this:
- Tener el progress bar con milestones en el overview porque ahi es donde vería cualquier usuario (sea o no del equipo) cómo va el proyecto y sus milestones
- En otra tab que solo puede ver los colaboradores del proyecto, la parte del index de milestones, y en donde pueden editarlo (probs cambie las vistas de create y edit a modal, y que de ahí mismo hagan el crud)

Does that make sense?

Reply:
Yes! Sounds great :) thanks!